pkgs/development/tools/database/liquibase/default.nix

```diff
@@ -13,6 +14,8 @@
 }:
 
 let
+  repoUrl = "https://github.com/liquibase/liquibase";
```

Comment:
Remove this.

```suggestion
```

Reply:
This is required because it uses `fetchurl` to get the release artifacts instead of `fetchFromGitHub`.

Reply:
(Used here)
```nix
  passthru.updateScript = gitUpdater {
    url = repoUrl;
```

Reply:
Also see here https://github.com/NixOS/nixpkgs/blob/af9aa4b566d719f8468746fde797dbde13c975df/pkgs/common-updater/git-updater.nix#L13

Reply:
This is not needed. You can simply write

```nix
  passthru.updateScript = gitUpdater {
    url = https://github.com/liquibase/liquibase";
```

Nix is not a macro language. Such "textual substitutions" are neither elegant nor useful.

Reply:
I was also using it in `src` to keep the repos in sync, but I won't push too hard on this. Pushed your suggestion.